test/src/e2e.dynval/doubleVote.test.ts

```diff
@@ -32,15 +33,15 @@ describe("Double vote detection", function() {
 
     const [alice, betty, charlie] = validators;
 
-    const { nodes } = withNodes(this, {
+    const { nodes, initialParams } = withNodes(this, {
         promiseExpect,
         validators: [
             { signer: alice, delegation: 5000, deposit: 10_000_000 - 0 },
             { signer: betty, delegation: 5000, deposit: 10_000_000 - 1 },
             { signer: charlie, delegation: 5000, deposit: 10_000_000 - 2 }
         ],
         overrideParams: {
-            termSeconds: 20,
+            termSeconds: 15,
```

Comment:
Why do you override it when they are not different with the default?

Reply:
I'll remove this one.